Functions.Templates/Templates/GenericWebHook-FSharp/metadata.json

```diff
@@ -1,11 +1,12 @@
 {
     "defaultFunctionName": "GenericWebhookFSharp",
-    "description": "$GenericWebhookFSharp_description",
+    "description": "$GenericWebhook_description",
     "name": "Generic Webhook",
```

Comment:
is webhook supposed to be capitalized?

Reply:
Yes it is for the resource name

Reply:
To clarify: this is only the name of the resource it is referencing. The description itself has webhook as lowercase.

Reply:
name property is visible, right? That should be "Generic webhook"

Reply:
OK sorry I thought you were commenting on the change.

Reply:
Yup I've changed the instances of webhook to lower case in all templates :)